st2api/st2api/controllers/v1/executionviews.py

```diff
@@ -49,9 +49,13 @@
 IGNORE_FILTERS = ['parent', 'timestamp', 'liveaction', 'trigger_instance']
 
 
+def csv(s):
+    return s.split(',')
+
+
 class FiltersController(RestController):
-    @jsexpose()
-    def get_all(self):
+    @jsexpose(arg_types=[csv])
+    def get_all(self, types=None):
```

Comment:
Would be to document `types` argument in the docstring.

Also, I didn't actually know you can reference query parameters in the same way you reference path arguments, I thought you needed to do `request.query` or something along those lines.

Besides that, LGTM.